core/trino-main/src/main/java/io/trino/execution/DropTableTask.java

```diff
@@ -20,6 +20,7 @@
 import io.trino.metadata.QualifiedObjectName;
 import io.trino.metadata.RedirectionAwareTableHandle;
 import io.trino.security.AccessControl;
+import io.trino.spi.TrinoException;
```

Comment:
changes to DropTableTask.java should ideally come together with SPI changes

they can be tested in isolation (without iceberg and delta connectors), using the mock connector